metabase/metabase
# Add dedicated docs pages for some expressions

Adds custom docs pages as specified here https://metaboat.slack.com/archives/C01MS7DQKR6/p1670426885693119?thread_ts=1664399343.985779&cid=C01MS7DQKR6

How to test:
- New -> Question -> Add custom column
- Type `datetimeAdd`
- Make sure the link to the docs goes to the dedicated page for that expression
- Same for `datetimeSubtract`, `datetimeDiff` (Postgres only), `convertTimezone` (Postgres only)


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/27163)
